src/main/java/uk/gov/register/indexer/function/RecordIndexFunction.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.register.indexer.function;
```

Comment:
Can we remove this? I think we're using `LatestByKeyIndexFunction.java` in place of this now. This is the same for `MetadataIndexFunction.java`, which seems to exist as an empty file - can we remove it?